### PR TITLE
refactor: Prepare using lifecycle for deprecations

### PR DESCRIPTION
**Pull request by krlmlr**
For #993.

### CODE REVIEW

**Comment by krlmlr** [comment]
@maelle: If this fails because of coverage, can you please add tests to this PR?

**Comment by maelle** [comment]
would do, but next week!

File: R/aaa-a-deprecate.R [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+#   along with this program; if not, write to the Free Software
+#   Foundation, Inc.,  51 Franklin Street, Fifth Floor, Boston, MA
+#   02110-1301 USA
+#
+###################################################################
+
+## For the future, right now, we do not warn or even message
+
+#' @importFrom utils packageName
+deprecated <- function(old, new) { # nocov start
```
Comment (maelle) [inline R/aaa-a-deprecate.R:26]: @krlmlr it seemed like a good idea to just not test this :sweat_smile:

**Comment by krlmlr** [comment]
Thanks!

**Pull request merged**